src/bindings/swig/python2/tests/testpy2_key.py

```diff
@@ -155,5 +167,15 @@ def test_iterator(self):
 		self.assertEqual(iter(k).value(),     "user")
 		self.assertEqual(reversed(k).value(), "c")
 
+	def test_helpers(self):
+            k = kdb.Key("user/noarray")
+            self.assertEqual(k.array_elements().index,    False)
```

Comment:
So .index is the way to ask if it is an array? What about providing `array_element().is` or similar for a True/False? Then this here could return None.

Reply:
I can add it but it feels unnecessary. You can check if it's an array style by checking any member of the namedtuple. I can also set `index` to None in the non-array-style case right away.

Reply:
> You can check if it's an array style by checking any member of the namedtuple.

Such statements are important for the tutorial so that it is clear how to use it.

> I can also set index to None in the non-array-style case right away.

Would be more consistent, wouldn't it?

Reply:
I could also throw an exception. Not sure if that's more handy though.

Reply:
ValueError it is